locales/de/common.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "F1 Kalender",
+  "subtitle": "Rennen, Qualifikation & Trainings",
+  "meta":{
+    "description": "Formel 1 Kalender für die Saison {{year}} mit allen Großen Preisen der Formel 1, Trainings &amp; Qualifikationen. Funktion für Erinnerungen. Alle Zeitzonen. Herunterladen oder abonnieren.",
+    "keywords": "F1, Formel 1, Rennzeiten, Rennen, Erinnerungen, Alarme, Große Preise, Großer Preis, Kalender, Daten, Startzeiten, Qualifikation, Training, London, Europa, {{year}}"
+  },
+  "options": {
+    "calendar": "Füge diese Renndaten - und Zeiten zu deinem Handy, Kalender, Outlook oder Google Calendar hinzu",
+    "timezonePicker": {
+      "showing": "Zeige die Zeiten für",
+      "pick": "Wähle eine Zeitzone...",
+      "button": "Erledigt"
+    }
+  },
+  "contribute": "Leiste einen Beitrag",
```

Comment:
'Beitragen' would be a better choice since it needs to fit in the language dropdown as far as I can see

Reply:
Yeah, didn't knew the context. Then 'Beitragen' is clearly the better choice.